setup.py

```diff
@@ -82,8 +82,8 @@ def build_extensions(self):
             opts.append(cpp_flag(self.compiler))
             if has_flag(self.compiler, '-fvisibility=hidden'):
                 opts.append('-fvisibility=hidden')
-            if has_flag(self.compiler, '-Werror'):
-                opts.append('-Werror')
+            #if has_flag(self.compiler, '-Werror'):
+            #    opts.append('-Werror')
```

Comment:
this shouldn't effect pio right, we're still compiling with -Werror there right?

Reply:
This doesn't effect PIO correct. The error was actually coming form pybind11 code and not ours as well loll So who really knows what's up.